pyaedt/modeler/Modeler.py

```diff
@@ -168,6 +192,67 @@ def rename(self, newname):
         return True
 
 
+class BaseLists(object):
+    """Base methods common to UserList.
+
+    Parameters
+    ----------
+    modeler :
+        Inherited parent object.
+    props : dict, optional
+        Dictionary of properties. The default is ``None``.
```

Comment:
I don't think you need this argument in your initializer.
```suggestion
    props : dict, optional
        Dictionary of properties. The default is ``None``.
```